src/bentoml/_internal/io_descriptors/image.py

```diff
@@ -196,17 +196,6 @@ def __init__(
                 f"Invalid Image mime_type '{mime_type}'; supported mime types are {', '.join(PIL.Image.MIME.values())} "
             )
 
-        for mtype in self._allowed_mimes:
```

Comment:
is this part expected?

Reply:
This will cause CI failure due to the differences between the client env and server env, when containerized.

From what I can tell `webp` support in PIL is optional:

```python
if SUPPORTED:
    Image.register_save(WebPImageFile.format, _save)
    if _webp.HAVE_WEBPANIM:
        Image.register_save_all(WebPImageFile.format, _save_all)
    Image.register_extension(WebPImageFile.format, ".webp")
    Image.register_mime(WebPImageFile.format, "image/webp")
```

So there are chances that the client env doesn't support webp but the server does, which will fail the check in the removed code block above.

Reply:
i see.